go-for-eat/screens/CreateEvent/CreateEvent.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Platform, View, Text } from 'react-native';
+import { Button } from 'react-native-elements'
+import { Constants, Location, Permissions } from 'expo';
+import DatePicker from 'react-native-datepicker'
+import { GooglePlacesAutocomplete } from '../../components/GooglePlacesAutocomplete';
+import { createEvent } from '../../actions';
+
+const moment = require('moment');
+
+import s from './styles';
+import cs from './customStyles';
+
+class CreateEvent extends Component {
+  constructor (props) {
+    super(props);
+    this.newEvent = {};
+  }
+
+  state = {
+    location: null,
+    errorMessage: null,
+    date: '',
+    time: '',
+  }
+
+  componentWillMount() {
+    if (Platform.OS === 'android' && !Constants.isDevice) {
+      this.setState({
+        errorMessage: 'Oops, this will not work on Sketch in an Android emulator. Try it on your device!',
+      });
+    } else {
+      this._getLocationAsync();
+    }
+  }
+
+  _getLocationAsync = async () => {
+    // console.log('GET LOCATION ASYNC');
+
+    let { status } = await Permissions.askAsync(Permissions.LOCATION);
+    if (status !== 'granted') {
+      this.setState({
+        errorMessage: 'Permission to access location was denied',
+      });
+    }
+
+    let location = await Location.getCurrentPositionAsync({});
+    this.setState({ location });
+    // console.log('this.setState', location);
+  };
+
+  handleGo = () => {
+    console.log('-----------------------handleGo---------------------');
+    if ( this.newEvent.place_id && this.state.date !== '' && this.state.time !== '') {
```

Comment:
remember that empty strings are falsy, this could also be && this.state.date && this.state.time